### PR TITLE
feat: add script handling

### PR DESCRIPTION
**Pull request by oleg-koval**
This Pull Request fulfills the following requirements:

- [x] The commit message follows our guidelines.
- [x] Tests for the changes have been added if needed.
- [x] Does not affect documentation or it has been added or updated.

Resolves #

### CODE REVIEW

File: src/optionsHandlers/handlers.ts [hunk 1]
```diff
@@ -27,3 +32,12 @@ export const handleDryRunFlag = (): { dryRun: boolean } => {
 };
 
 export const handleDebugFlag = (): boolean => getInput(Flags.debug) === 'true';
+
+export const handleScriptPathFlag = async (): Promise<string> => {
+  const scriptPathInput: string = getInput(Flags.scriptPath);
```
Comment (merlinnot) [inline src/optionsHandlers/handlers.ts]: Should we resolve the path relative to `GITHUB_WORKSPACE`?

File: README.md [hunk 2]
```diff
@@ -29,6 +29,9 @@ Add a workflow file to your repository to create custom automated processes.
   and `success` steps. If not provided defaults to false.
 - DEBUG: [Optional] Boolean value expected. Outputs debug information for
   semantic-release plugins. If not provided, defaults to false.
+- SCRIPT_PATH: [Optional] String value expected. Executes script with
+  @semantic-release/exec plugin. Script needs to be placed in "script" directory
```
[the remark this answers is not in the document]
Comment (oleg-koval) [reply README.md]: That was a typo in this line, in code expected directory is `scripts`.
We can provide a relative path too, the idea was to have it consistent with other repos. I can change it to be fully custom, WDYT?

File: src/optionsHandlers/handlers.ts (hunk 1 above)
Comment (oleg-koval) [reply src/optionsHandlers/handlers.ts]: Actions and shell commands execute in `/github/workspace` (GITHUB_WORKSPACE) directory, so `fs.exists` is able to check if the required file is present in the workspace.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (merlinnot) [reply README.md]: I think we will keep it consistent in repos of @ridedott, but other users might want to have it elsewhere. I don't think we should enforce it on the community.

[the remark this answers is not in the document]
Comment (oleg-koval) [reply README.md]: Agree!

File: README.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ Add a workflow file to your repository to create custom automated processes.
   and `success` steps. If not provided defaults to false.
 - DEBUG: [Optional] Boolean value expected. Outputs debug information for
   semantic-release plugins. If not provided, defaults to false.
-- SCRIPT_PATH: [Optional] String value expected. Executes script with
+- SCRIPTS_PATH: [Optional] String value expected. Executes script with
```
Comment (merlinnot) [inline README.md]: I think that the previous `SCRIPT_PATH` was actually correct, as you can only provide one script here. My previous comment was in regards to `"script" directory`, which in our case is always `"scripts"`, but we also just aligned to remove this limitation. Sorry for the confusion.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (oleg-koval) [reply README.md]: Actually nothing needs to be changed in code :) Documentation just needs to reflect this.

**Pull request merged**